packages/web-client/app/components/card-pay/layer-one-connect-card/index.hbs

```diff
@@ -6,12 +6,9 @@
   data-test-mainnnet-connection-action-container
   ...attributes
 >
-  <section class="layer-one-connect-card__body">
-    {{#if this.connectedWalletProvider}}
-      <header aria-label="Layer 1 Connected Header" class="layer-one-connect-card__body-header" data-test-mainnet-connection-header>
-        {{this.connectedWalletProvider.name}}
-        <img class="layer-one-connect-card__body-header-logo" src={{this.connectedWalletProvider.logo}} alt="" role="presentation">
-      </header>
+  {{#if this.connectedWalletProvider}}
+    <ActionCardContainer::Section @title={{this.connectedWalletProvider.name}} class="layer-one-connect-card__body">
+      <img class="layer-one-connect-card__logo" src={{this.connectedWalletProvider.logo}} alt="MetaMask" width="60" height="60">
```

Comment:
I think there isn't much value in adding alt text to these logos. It is meant to be an accompaniment to the text, which already should have the same value. (this shouldn't be hardcoded).